dojo/tools/trufflehog/parser.py

```diff
@@ -166,7 +168,7 @@ def get_findings_v3(self, data, test):
                     severity = "Medium"
 
             dupe_key = hashlib.md5(
```

Comment:
Maybe it is better to use a DEDUPE_ALGO_UNIQUE_ID_FROM_TOOL with unique_id_from_tool here?